app/components/ak-button/index.scss

```diff
@@ -0,0 +1,29 @@
+.btn {
+  height: 2.5rem;
+  padding: 0.57em 0.85em;
```

Comment:
this padding number is so weird. where is this coming from?
also height should be em not rem as it should depend on the font size of the text used not root font size

Reply:
1. was tried to derive it, but having 0.5em is better 👍 
2. updated height unit